server.go

```diff
@@ -43,6 +45,17 @@ func main() {
 	reqLog := flag.Bool("reqlog", false, "Enables request logging -- do NOT use in production")
 	flag.Parse()
 
+	err := os.Mkdir("/etc/ielogs", 0755)
```

Comment:
I would prefer that the location be configurable via a flag / env var (see how I support both with other options using `getConfigValue` in the _integrator_ project).  If no option is passed in (via flag or env variable), then it should just default to whatever log normally defaults to (STDOUT/STDERR?).

I suggest this because I don't want stuff writing to `/etc` on my development machine, and in production, folks will probably want to configure it anyway (they may not allow programs to write to `/etc`).